hermes_nemisis/calibration/calibration.py

```diff
@@ -183,23 +197,434 @@ def l0_sci_data_to_cdf(data: dict, original_filename: Path) -> Path:
     >>> data_filename = Path("hermes_NEM_l0_2022339-000000_v0.bin")
     >>> metadata = parse_science_filename(data_filename)  # doctest: +SKIP
     >>> data_packets = calib.parse_l0_sci_packets(data_filename)  # doctest: +SKIP
-    >>> cdf_filename = calib.l0_sci_data_to_cdf(data_packets, data_filename)  # doctest: +SKIP
+    >>> cdf_filename = calib.l0_sci_data_to_l1(data_packets, data_filename)  # doctest: +SKIP
     """
+    log.info(f"Begin l0_sci_data_to_l1")
+
     file_metadata = parse_science_filename(original_filename.name)
 
-    cdf_filename = original_filename.parent / create_science_filename(
-        file_metadata["instrument"],
-        file_metadata["time"],
-        "l1",
-        f'1.0.{file_metadata["version"]}',
+    cdf_version = f'1.0.{int(file_metadata["version"])}'
```

Comment:
I believe all the files should already contain an `X.Y.Z` version. I know the test file in this PR does not match this, but will the files we receive from the MOC have the same versioning scheme @dbarrous  @ehsteve ?